src/modules/registrypreview/RegistryPreviewUILib/RegistryPreviewMainPage.xaml

```diff
@@ -10,6 +11,25 @@
     xmlns:ui="using:CommunityToolkit.WinUI"
     mc:Ignorable="d">
 
+    <Page.Resources>
+        <ResourceDictionary>
+            <converter:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
+
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <SolidColorBrush x:Key="HexBox_SelectionTextBrush" Color="{ThemeResource TextOnAccentFillColorSelectedText}" />
+                    <SolidColorBrush x:Key="HexBox_SelectionBackgroundBrush" Color="{ThemeResource SystemAccentColor}" />
+                    <SolidColorBrush x:Key="HexBox_AlternatingDataColumnTextBrush" Color="{ThemeResource TextFillColorSecondary}" />
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <SolidColorBrush x:Key="HexBox_SelectionTextBrush" Color="{ThemeResource SystemColorHighlightTextColor}" />
+                    <SolidColorBrush x:Key="HexBox_SelectionBackgroundBrush" Color="{ThemeResource SystemColorHighlightColor}" />
+                    <SolidColorBrush x:Key="HexBox_AlternatingDataColumnTextBrush" Color="{ThemeResource SystemColorGrayTextColor}" />
```

Comment:
Wondering if we should do this? 🤔 Or if we should keep all columns in text color for High Contrast? (Feedback from accessibility team is welcome.)

Reply:
Found this in guidelines:
_"Do not use SystemColorGrayTextColor for body copy that is secondary or acts as hint text. This is intended for disabled content only."_